cmd/process-user-data/types.go

```diff
@@ -10,6 +10,10 @@ const (
 	AWSUserDataImdsUrl   = "http://169.254.169.254/latest/user-data"
 
 	defaultAgentConfigPath = "/etc/agent-config.toml"
+
+	authJsonDirPath    = "/etc/attestation-agent/"
```

Comment:
nit: The auth.json is an image-rs config now, so I'm not sure if `/etc/attestation-agent/` is the best directory rather than just `/etc/`, but it's not a blocker

Reply:
I can change the directory. Thought `attestation-agent` might be a good place to keep all these auth stuff. I'll change to `/etc` to align with the agent-config.toml location. In future we can decide to put all these configs under `/etc/kata` may be.

Reply:
Yeah, I wonder if, as part of the merge into main, we might end up adopting policies to set some of this config rather than the current approach anyway.